tests/functional/u/undefined/undefined_variable_py38.py

```diff
@@ -123,7 +123,7 @@ def type_annotation_used_after_comprehension():
 
 def type_annotation_unused_after_comprehension():
     """https://github.com/PyCQA/pylint/issues/5326"""
-    my_int: int  # [unused-variable]
+    my_int: int
```

Comment:
This message was added in #5711 without much conviction. It probably shouldn't be here at all. Removing this is a better fix for the issue in #5326, which is a ticket I opened to say _hey we started emitting a message here!_